tests/fixtures/saas/simon_data_fixtures.py

```diff
@@ -0,0 +1,69 @@
+from typing import Any, Dict, Generator
+
+import pydash
+import pytest
+
+from tests.ops.integration_tests.saas.connector_runner import (
+    ConnectorRunner,
+    generate_random_email,
+)
+from tests.ops.test_helpers.vault_client import get_secrets
+
+secrets = get_secrets("simon_data")
+
+
+@pytest.fixture(scope="session")
+def simon_data_secrets(saas_config) -> Dict[str, Any]:
+    return {
+        "domain": pydash.get(saas_config, "simon_data.domain") or secrets["domain"],
+        "api_key": pydash.get(saas_config, "simon_data.api_key") or secrets["api_key"],
+    }
+
+
+@pytest.fixture(scope="session")
+def simon_data_identity_email(saas_config) -> str:
+    return (
+        pydash.get(saas_config, "simon_data.identity_email")
+        or secrets["identity_email"]
+    )
+
+
+@pytest.fixture
+def simon_data_erasure_identity_email() -> str:
+    return generate_random_email()
+
+
+@pytest.fixture
+def simon_data_external_references() -> Dict[str, Any]:
+    return {}
+
+
+@pytest.fixture
+def simon_data_erasure_external_references() -> Dict[str, Any]:
+    return {}
```

Comment:
You can delete this fixture, `simon_data_external_references`, and remove them as parameters in the `ConnectorRunner` below. Kinda clunky but in terms of the SaaS template nox command it's better to have them and not need them than to need them and not have them.